libs/test-common/src/main/java/org/codice/ddf/test/common/features/FeatureUtilities.java

```diff
@@ -69,7 +69,8 @@
     List<String> featureNames = new ArrayList<>();
 
     try (FileInputStream fi = new FileInputStream(new File(featureFilePath))) {
-      Document featuresFile = DocumentBuilderFactory.newInstance().newDocumentBuilder().parse(fi);
+      Document featuresFile =
+          XMLUtils.getInstance().getSecureDocumentBuilderFactory().newDocumentBuilder().parse(fi);
```

Comment:
✏️ This could be simplified to `XMLUtils.getInstance().getSecureDocumentBuilder(false).parse(fi)`